src/aux/InsphereCalculator.cpp

```diff
@@ -0,0 +1,165 @@
+#include "InsphereCalculator.h"
```

Comment:
I guess you don't use the one from seissol:
https://github.com/SeisSol/SeisSol/blob/dd018b3398258a23ec2a33c74bd7f31b503dcca6/src/Initializer/time_stepping/GlobalTimestep.cpp#L22-L32
because you want to avoid another dependency?

Reply:
Yup—that was the main thing. Though it probably wouldn't be a big one anyways... I.e. it's thinkable I'd say.

IIRC I copied the calculation from there.